src/renderer/components/ft-slider/ft-slider.js

```diff
@@ -39,7 +39,7 @@ export default Vue.extend({
       if (this.valueExtension === null) {
         return this.currentValue
       } else {
-        return `${this.currentValue}${this.valueExtension}`
+        return `${this.currentValue} ${this.valueExtension}`
```

Comment:
This is the one change that I'm not so sure with.  Judging from what's included in the language that you're using, this is the preferred formatted for your language, though for most other languages, the expected formatting is how it currently is.  Let's revert this back for now, and I will create a solution at a later point.  There are many languages I have to account for and I'll likely open a general issue for finding and fixing all of these so that they look correct in all languages.

Reply:
I'm sorry to say, but you are wrong. It actually is the standard way of writing. Have a look at https://en.wikipedia.org/wiki/ISO_31-0, in the section **Typographic conventions**, subsection **Expressions.**

Of course if you definitely do not want to use it, I will change it (after all it's your project, so obviously you can decide). Please reconsider and let me know your final decision. Thank you.

Reply:
And I will leave you with [this reference](https://en.wikipedia.org/w/index.php?title=Percent_sign#Form_and_spacing) as I stand by my original statement.  The formatting is different for several different languages and needs to be accounted for on a case by case basis.  This is something that goes above just this situation and is likely going to need it's own development cycle to get everything.

For now, I'd like to get a thread going that compiles all of these different edge cases together so that when we get to it, it'll be much easier to get these all resolved in one go.  Let's keep it the way it is for the time being and we will take a closer look at it once we get a good list put together.